source/c_pwm.c

```diff
@@ -94,6 +94,17 @@ void export_pwm(struct pwm_exp *new_pwm)
     }
 }
 
+int is_dmtimer_pin(pwm_t *p) {
+    if(strlen(p->module) < 5)
```

Comment:
Do we need the length check? I think `strcmp` will do something sane even if `p->module` is short, no?

Reply:
That's a good point -- I originally included the length check for `strncpy`, but it looks like `strncpy`'s behavior will still be [well-defined](http://www.cplusplus.com/reference/cstring/strncpy/) without this check. I've removed this conditional in af9ea51.